src/signals/incident/components/form/FileInput/index.js

```diff
@@ -64,32 +131,43 @@ const FileInput = ({ handler, touched, hasError, getError, parent, meta, validat
             hasError={hasError}
             getError={getError}
           >
-            {handler().value ?
-              <div className="file-input__preview">
-                <button
-                  title="Verwijder upload foto"
-                  className="file-input__button-delete link-functional delete"
-                  onClick={(e) => handleClear(e, handler().value)}
-                />
-                {fileType && fileType.split('/')[0] === 'image' ?
-                  <img
-                    alt="Preview uploaded foto"
-                    src={handler().value}
-                    className="file-input__preview-image"
-                  />
-                : ''}
-              </div>
-            :
-              <div className="invoer">
+            <div className="file-input">
+              {previews.length ? previews.map((preview) =>
+                (<div key={preview} className={`file-input__preview ${preview.includes('loading') ? 'file-input__preview--loading' : ''}`}>
+                  {preview.includes('loading') ?
+                    <div className="progress-indicator progress-red"></div>
+                  :
+                    <div style={{ backgroundImage: `URL(${preview})` }} className="file-input__preview-image">
+                      <button title="Verwijder deze foto" className="file-input__preview-button-delete" onClick={(e) => removeFile(e, preview, previews, handler().value)} />
+                    </div>
+                 }
+                </div>)
+              ) : '' }
+
+              {previews.length < maxNumberOfFiles ?
+              (<div className="file-input__button">
+                <label htmlFor="formUpload" className="file-input__button-label">
+                  <div htmlFor="formUpload" className="file-input__button-label-icon">&nbsp;</div>
```

Comment:
What is the point in having a label when it's text content is a non-breaking string instead of a descriptive label?

Reply:
the empty div is the icon